bftengine/include/bftengine/ReplicaConfig.hpp

```diff
@@ -65,6 +65,7 @@ class ReplicaConfig : public concord::serialize::SerializableFactory<ReplicaConf
                "1 <= concurrencyLevel <= 30");
   CONFIG_PARAM(viewChangeProtocolEnabled, bool, false, "whether the view change protocol enabled");
   CONFIG_PARAM(blockAccumulation, bool, false, "whether the block accumulation enabled");
+  CONFIG_PARAM(adaptiveConsensus, bool, true, "whether the consensus batch size adaptive feature is enabled");
```

Comment:
Actually, I would add a new policy for this feature. Something like BATCH_ADAPTIVE.

Reply:
Added